lib/node/runner/node_runner.go

```diff
@@ -149,6 +149,9 @@ func (nr *NodeRunner) Ready() {
 		nr.Conf,
 	)
 
+	nr.network.AddMiddleware(network.RouterNameAPI, network.RecoverMiddleware(false))
```

Comment:
There are `-verbose` option in `cmd/sebak/cmd/run.go`, we can use it for`printStack`.

Reply:
Ok, I got it. :-)